.circleci/config.yml

```diff
@@ -76,7 +76,6 @@ jobs:
           command: |
             pip install numpy cython pybind11 pythran ninja meson
             pip install -r requirements/doc.txt
-            pip install "myst-nb<1.0.0"
```

Comment:
So it gets installed transitively right?

Reply:
Yes (although I didn't realise this at first hence all the force pushes!)

https://github.com/scipy/scipy/blob/2c6915ad7361d814621596afb12a4dd78603f77c/requirements/doc.txt#L9